heat/core/tests/test_suites/basic_test.py

```diff
@@ -1,4 +1,5 @@
 from unittest import TestCase
+import os
```

Comment:
This whole file still needs an __init__.py

Reply:
init.py is no longer needed for Python 3.3+

Reply:
Or what do you mean?

Reply:
You are right, it is not needed anymore. However, we have the init.pys in all the folders. I think we should be consistent here.